tests/x11/pidgin/pidgin_IRC.pm

```diff
@@ -40,7 +40,7 @@ sub run {
     assert_and_click 'pidgin-irc-account';
 
     # Warning of spoofing ip may appear
-    assert_screen([qw(pidgin-spoofing-ip pidgin-irc-sledtesting)]);
+    assert_screen("pidgin-spoofing-ip", 3);
```

Comment:
Please do not use a timeout less than default. Also, you are deleting the second alternative so the next line with the `match_has_tag` is useless.

Reply:
@okurz Thanks for feedback, there are good ideas in it. This is first PR by @msmeissn and will polish it together.

Reply:
Na, @msmeissn already provided fixes in before to this repo :) Still, don't take my comments as too harsh. I just want to make sure that in the current way the PR is not merged because I can only see it causing very flaky unstable tests and this is IMHO the biggest trust issue that people have with openQA

Reply:
I re-checked logic sequence of the test and I think it is not correct in `assert_screen([qw(pidgin-spoofing-ip pidgin-irc-sledtesting)])`. There is a chance of `pidgin-spoofing-ip`, which may occur (but don't have to). But why is there match for `pidgin-irc-sledtesting`? This shouldn't be there, because this match is supposed to be done little bit later. I think @msmeissn had good idea to remove it and there should be `if (check_screen('pidgin-spoofing-ip')) {` instead of `assert_screen` with `match_has_tag`.  SImilar incorrect logic is also for CTCP check.
I did a test run  with `check_screen` and it looks fine http://10.100.12.105/tests/226#.